test/forms/inputs/icon-input.test.js

```diff
@@ -9,9 +9,9 @@ const props = { input: { name, value, onChange }, meta: {}, icon: 'foo' }
 
 test('IconInput adds class "icon-label" to surrounding fieldset', () => {
```

Comment:
I believe the test description needs to be rewritten not to directly reference `fieldset`